ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -172,6 +172,45 @@ message DamlTransactionEntry {
 
   // The time used to derive contract ids
   google.protobuf.Timestamp submission_time = 6;
+
+  // The pre-computed transaction blinding information. Optional.
+  DamlTransactionBlindingInfo blinding_info = 7;
+}
+
+// A transaction's blinding information, consisting of disclosure and
```

Comment:
Instead of copying the code documentation from `BlindingInfo` please just reference that class from here.

Reply:
Done.